src/GameInternal.cpp

```diff
@@ -333,21 +304,25 @@ bool GameInternal::isRunning() const
 	return running;
 }
 
-void GameInternal::setRunning(bool running)
+void GameInternal::setRunning(bool running) //TODO: might be depracted
```

Comment:
Used in https://github.com/VEGO-Engine/chickengame/blob/e4a328d0d6dc16e7f804c45bd6db9b6632001546/src/Chickengame.cpp#L155